packages/ember-data/lib/serializers/json-api-serializer.js

```diff
@@ -26,6 +26,27 @@ export default JSONSerializer.extend({
   isNewSerializerAPI: true,
 
   /*
+    @method _normalizeDocumentHelper
+    @param {Object} documentHash
+    @return {Object}
+    @private
+  */
+  _normalizeDocumentHelper: function(documentHash) {
```

Comment:
It just felt wrong (no science here) to call `_normalizeResponse` with a lot of empty (or faked) parameters and there were already a couple of normalize-helpers so I figured why not.